src/Utilities/Task.cs

```diff
@@ -99,6 +99,11 @@ protected Task(ResourceManager taskResources, string helpKeywordPrefix)
         /// </summary>
         public IBuildEngine8 BuildEngine8 => (IBuildEngine8)BuildEngine;
 
+        /// <summary>
+        /// Retrieves the <see cref="IBuildEngine9" /> version of the build engine interface provided by the host.
+        /// </summary>
+        public IBuildEngine9 BuildEngine9 => (IBuildEngine9)BuildEngine;
```

Comment:
As I mentioned in the meeting, I slightly prefer reusing IBuildEngine8, but this is fine.